extensions/Push/lib/Connector/Webhook.pm

```diff
@@ -96,12 +97,23 @@ sub send {
 
     my $bug_data   = $self->_get_bug_data($payload);
     my $is_private = $bug_data->{is_private};
+
+    if ($message->routing_key eq 'comment.create'){
+      my $comment->{field} = $payload->{comment}->{number} == 0 ? 'description' : 'comment';
+      #$comment->{added}    = HTML::Strip->new()->parse($payload->{comment}->{body});
+      $comment->{added}    = $payload->{comment}->{body};
```

Comment:
There is no need do strip HTML from the comment as we are sending the raw text and not the fully formatted text you see in the web UI. quoteUrls() in the code is what changes URLs into clickable links, etc in the web UI and we do not need that for JSON data.

Also if we can somehow load the Bugzilla/Comment.pm object for the comment that is added, we can call `$comment->body_full` which will give us extra text such as `Attachment NNNN created` or `*** This bug has been marked as a duplicate of NNNN ***`.